.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -34,15 +34,15 @@ vcgencmd bootloader_config
 
 **SD card boot (please complete the following information):**  
  - SD card type
- - Partition information (sudo fdisk -l) if you are able to obtain this from another computer.
+ - Partition information (`sudo fdisk -l`) if you are able to obtain this from another computer.
 
 **USB boot (please complete the following information):**  
-Verify that the the USB device works correctly when hot-plugged under Linux and attache the output of 'lsusb -vvv'
+Verify that the the USB device works correctly when hot-plugged under Linux and attach the output of 'lsusb -vvv'
```

Comment:
This is an instruction, not a request for information. It is also a senence, and so requires a `.`.